README.md

```diff
@@ -1,20 +1,157 @@
-[![Build Status](https://img.shields.io/azure-devops/build/responsibleai/fairlearn/6/dev?failed_label=bad&passed_label=good&label=GatedCheckin%3ADev)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=1&branchName=dev) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![pypi badge](https://img.shields.io/badge/pypi-0.2.0-blue)
+[![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
-# Fairness in Machine Learning - Mitigation algorithms
 
-A Python package that implements a variety of fairness-related algorithms to mitigate bias including:
+
+Fairlearn
+=============================================================
+Fairlearn is a machine learning fairness toolkit that makes it easier for anyone involved in the development of Artificial Intelligence (AI) systems to monitor and understand the impact of unfairness in their machine learning lifecycle, and use state-of-the-art techniques to mitigate the observed unfairness. Fairlearn repository contains an SDK and Jupyter notebooks with examples to showcase its use.
+
+- [Overview of Fairlearn](#intro)
+- [Target Audience](#target)
+- [Fairness Assessment Dashboard](#getting-started)
+- [Unfairness Mitigation Techniques](#models)
+- [Install Fairlearn](#install)
+- [Contributing](#contributing)
+- [Code of Conduct](#code)
+- [Build Status](#build-status)
+- [Additional References](#refs)
+
+
+
+ 
+# <a name="intro"></a>
+# Overview of Fairlearn
+
+We are at an age where many processes and applications have become or are becoming automated by AI systems. AI is the product of human processes and decisions used to create it, the data used to train it, and the environment used to test it.  The AI system can exhibit different, sometimes harmful, behaviors as a result of this process and systematically discriminate against people based on certain attributes protected by law (e.g., gender, ethnicity). Therefore, regulated industries, or any other industry with legal, moral and ethical responsibilities, need tools to ensure the fairness of models and the predictions they produce. 
+
+
+`Fairlearn` is an open source unfairness assessment and mitigation toolkit that provides business stakeholders, executives, developers, and data scientists with insights into the unfairness of their model predictions, and techniques to mitigate that. In particular, Fairlearn assists in 
+
+1) Raising awareness of inherent unfairness in model predictions among those involved in developing AI applications by demonstrating harms visited on vulnerable groups
```

Comment:
Is 'harm' too loaded a word to use here? We show how predictions can be unfair; the extent to which that's a harm is a slightly different question.